web/components/contract/contract-overview.tsx

```diff
@@ -560,6 +565,24 @@ const ChoiceOverview = (props: {
     [answers, resolutions, shouldAnswersSumToOne, sort]
   )
 
+  useEffect(() => {
+    if (
+      currentUserId &&
```

Comment:
I think this logic is cursed, should be `currentUserId && (case1 || case2 || case3)`